### PR TITLE
reconstruction script and fixes

### PR DESCRIPTION
**Pull request by evan-gray**

### CODE REVIEW

File: watcher/scripts/reconstructVAA.ts [hunk 1]
```diff
@@ -0,0 +1,297 @@
...
+  { chain: 11, txHash: '0x36707fcbd9136cda044ed178c70fea40c233855ac4d06b810bef3c0b23e2fbe8' },
+  { chain: 11, txHash: '0x10daa711167a0333877bc29e53c240851f2aeb27917c9e980891766ba80d45b4' },
+  { chain: 11, txHash: '0x922357276f1e116c5e9249035677ac2ec9b1a0fd31cc0cfa653563991f75fe8c' },
+  { chain: 11, txHash: '0xbb1466fa50a5d2d6d9342a84800a99a74900c6942b4b4dbd22c7b8634396365a' },
+  { chain: 11, txHash: '0xc06b5041d402531c991befde448d815ebf327b4ff76eaea1bf8e645fe9ffaca4' },
+  { chain: 11, txHash: '0xa75a8fcbf135384420c0780a90f59edcbb3159bbb2b80dc100204a61b8f221bf' },
+  { chain: 11, txHash: '0x48f532fdc933fc27afb73c580ade371e2d961dee8e56b496298f4bc4dddd19dc' },
+  { chain: 11, txHash: '0xd16540dcddea5a467ecb5c100329e86495719f68d7b01d883379eaea2c2746ef' },
+  { chain: 12, txHash: '0x531f78064f97c08875fb8e28054f7a52ea117440327a9ad8c4c9a0d091897098' },
+  { chain: 12, txHash: '0x436a6676477dd90f0cf0e8a0e78126c6653c7fe2e44a5dc921dc1fa3d5c4853b' },
```
Comment (panoel) [inline watcher/scripts/reconstructVAA.ts:39]: Do we add this chain/hash es each time?  Or do we have this read in a file?

File: watcher/scripts/reconstructVAA.ts [hunk 2]
```diff
@@ -0,0 +1,297 @@
...
+];
+
+type GuardianSetEntry = { pubkey: string; name: string };
+
+// https://github.com/wormhole-foundation/wormhole-networks/blob/master/mainnetv2/guardianset/v3.prototxt
+const gs3: GuardianSetEntry[] = [
```
Comment (panoel) [inline watcher/scripts/reconstructVAA.ts:45]: Will this be used any where else and should be in a const file?

File: watcher/scripts/reconstructVAA.ts [hunk 3]
```diff
@@ -0,0 +1,297 @@
...
+    pubkey: '0x5E1487F35515d02A92753504a8D75471b9f49EdB',
+    name: 'Triton',
+  },
+  {
+    pubkey: '0x6FbEBc898F403E4773E95feB15E80C9A99c8348d',
+    name: 'Staking Facilities',
+  },
+];
+
+function serialiseSignature(sig: Signature): string {
```
Comment (panoel) [inline watcher/scripts/reconstructVAA.ts:124]: We are British?

File: watcher/scripts/reconstructVAA.ts [hunk 4]
```diff
@@ -0,0 +1,297 @@
...
+    ...vaa.signatures.map((sig) => serialiseSignature(sig)),
+    vaaBody(vaa),
+  ];
+  return body.join('');
+}
+
+(async () => {
+  for (const miss of misses) {
+    const rpc = RPCS_BY_CHAIN[coalesceChainName(miss.chain)];
+    if (!isEVMChain(miss.chain)) {
```
Comment (panoel) [inline watcher/scripts/reconstructVAA.ts:158]: Should we write an issue to support non-EVM chains?

File: common/src/consts.ts [hunk 5]
```diff
@@ -23,5 +23,5 @@ export const INITIAL_DEPLOYMENT_BLOCK_BY_CHAIN: {
   aptos: '0', // block is 1094390 but AptosWatcher uses sequence number instead
   near: '72767136',
   xpla: '777549',
-  solana: '94401321', // https://explorer.solana.com/tx/KhLy688yDxbP7xbXVXK7TGpZU5DAFHbYiaoX16zZArxvVySz8i8g7N7Ss2noQYoq9XRbg6HDzrQBjUfmNcSWwhe
+  solana: '94396403', // https://solscan.io/tx/2L8rQY94W2d44sycRkhHA1PyXdh5z6ND541ftDDk1dgBcv6RLR9a3zUgTJispPmXjkmqdqd5EDytXcnP5PC2AmEJ
```
Comment (panoel) [inline common/src/consts.ts:26]: I got a 429 when I tried going to this URL.   Whomp, whomp.

**Review by panoel — APPROVED** [review]

**Review by panoel — DISMISSED** [review]

**Comment by evan-gray** [comment]
https://github.com/wormhole-foundation/wormhole-dashboard/pull/59

**Pull request closed without merging**